server/odc-service/src/main/java/com/oceanbase/odc/service/websocket/OBClientReadThread.java

```diff
@@ -28,6 +29,8 @@
 
 @Slf4j
 public class OBClientReadThread extends Thread {
+    private static final Pattern UNEXPECTED_SEQUENCE_PATTERN = Pattern.compile(" \\r");
```

Comment:
what is the white space of this pattern used for? `\\s`?

Reply:
white space may not  affect line breaking behavior

Reply:
` \\r` may change to `\\r`

Reply:
We should not change this , because the obclient error sequences are always with a blank space~
![image](https://github.com/oceanbase/odc/assets/36879124/276f8a0b-7671-4757-96cf-6aef6c2f46be)